docs/src/repeated.md

```diff
@@ -0,0 +1,150 @@
+
+The CO₂ uptake of six plants from Quebec and six plants from Mississippi was measured at several levels of ambient CO₂ concentration.
+Half the plants of each type were chilled overnight before the experiment was conducted.
+
+#### Source
+Potvin, C., Lechowicz, M. J. and Tardif, S. (1990) “The statistical analysis of ecophysiological response curves obtained from experiments involving repeated measures”, *Ecology*, *71*, 1389-1400.
+
+```@example repeated
+data(dataset(:CO2)) * mapping(
+    :conc => "Ambient CO₂ concentration [mL/L]",
+    :uptake => "CO₂ uptake rate [μmol/m²s]",
+    row = :treatment,
+    col = :type,
+    color = :plant,
+) |> draw
+```
+
+## DNase
+```@example repeated
+dataset(:DNase)
+```
+Development of an ELISA assay for the recombinant protein DNase in rat serum
+- `density`: measured optical density [dimensionless]
+- `conc`: concentration of the protein
+- `run`: assay run
+
+#### Source
+Davidian, M. and Giltinan, D. M. (1995) *Nonlinear Models for Repeated Measurement Data*, Chapman & Hall (section 5.2.4, p. 134)
+
+```@example repeated
+let dnase = dataset(:DNase)
+    r = renamer([s => s for s in levels(dnase.run)])
+    plt = data(dnase) *
+        mapping(
+            :conc => "Concentration of the protein DNase",
+            :density => "Measured optical density",
+            marker=(:run => r => "Run"),
+            color=(:run => r => "Run"),
+        ) *
+    (visual(Lines) + visual(Scatter))
+    draw(plt)
+end
+```
+## Dialyzer
+```@example repeated
+dataset(:Dialyzer)
+```
+- `rate`: the hemodialyzer ultrafiltration rate [mL/hr]
+- `subject`: indicator of the subject on which the measurement was made
+- `pressure`: transmembrane pressure [dmHg]
+- `qb`: bovine blood flow rate [dL/min]
+- `index`: within-subject index of observation
+
+Vonesh, E. F. and Carter, R. L. (1992), *Mixed-effects nonlinear regression for unbalanced repeated measures*, _Biometrics_, *48*, 1-18
+describe data on high-flux hemodialyzers to asses their *in-vivo* ultrafiltration characteristics.
+
+The ultrafiltration rates (in mL/hr) of 20 high-flux dialyzers were measured at seven different transmembrane pressures (in dmHg).
+The _in vitro_ evaluation of the dialyzers used bovine blood at flow rates of either 200 dl/min or 300 dl/min.
+
+```@example repeated
+let dial = dataset(:Dialyzer)
+    s = renamer(([s => s for s in levels(dial.subject)]))
+    plt = data(dataset(:Dialyzer)) *
+        mapping(
+            :pressure => "Transmembrane pressure [dmHg]",
+            :rate => "Hemodialyzer ultrafiltration rate [mL/hr]",
+            color=:subject => s => "Subject",
+            marker=:subject => s => "Subject",
+            col = :qb,
+        )
+    plt * (visual(Lines) + visual(Scatter)) |> draw
+end
+```
+
+## Rail
+
+```@example repeated
+dataset(:Rail)
+```
+- `travel`: measurements travel time for ultrasonic head-waves in a rail [ns].  The value recorded is the original travel time minus 36,100 ns.
```

Comment:
I'm guessing the offset is because of equipment latency, but is this stated in the original source?

Reply:
It may have just been to center the data approximately so that the observed changes are not so small relative to the measurement values.

Reply:
I was just curious, but that also makes sense!